backend/Repositories/SightingRepo.cs

```diff
@@ -24,8 +25,17 @@ public SightingRepo(WhaleSpottingDbContext context)
         public IEnumerable<Sighting> GetAllSightings()
         {
             return _context
-                .Sightings
-                .Include(s => s.Species);
+                    .Sightings
+                    .Include(s => s.Species);
```

Comment:
Please un-indent these two lines by one level!
```suggestion
                .Sightings
                .Include(s => s.Species);
```